client/tests/components.test.js

```diff
@@ -14,3 +14,10 @@ describe('Nav', () => {
 	});
 });
 
+import SigninForm from '../src/components/SigninForm';
+
+describe('SigninForm', () => {
```

Comment:
Unit test needs to be build out further for completeness...it should test out cases such as but not limited to:

- access is not granted with invalid credentials
- registration doesn't allow for new user with same email address
  - even if exact valid credentials provided no access should be granted via registration of existing user email
- create a token with shorter expiration...log-in to confirm access within timeframe, wait sufficient time, attempt to access user information but fails because cookie has expired, and checks redirects to sign-in page
...